lib/chef/resource/powershell_package_source.rb

```diff
@@ -24,36 +25,131 @@ class PowershellPackageSource < Chef::Resource
 
       provides :powershell_package_source
 
-      description "Use the **powershell_package_source** resource to register a PowerShell package repository."
+      description "Use the **powershell_package_source** resource to register a PowerShell Repository or other Package Source type with. There are 2 distinct objects we care about here. The first is a Package Source like a PowerShell Repository or a Nuget Source. The second object is a provider that PowerShell uses to get to that source with, like PowerShellGet, Nuget, Chocolatey, etc. "
       introduced "14.3"
+      examples <<~DOC
+        **Add a new PSRepository that is not trusted and which requires credentials to connect to**:
+
+        ```ruby
+        powershell_package_source 'PowerShellModules' do
+          source_name                  "PowerShellModules"
+          source_location              "https://pkgs.dev.azure.com/some-org/some-project/_packaging/some_feed/nuget/v2"
+          publish_location             "https://pkgs.dev.azure.com/some-org/some-project/_packaging/some_feed/nuget/v2"
+          trusted                      false
+          user_name                    "someuser@somelocation.io"
```

Comment:
In our other resources we use 'user' and 'password'. We should be consistent here.

Reply:
corrected